doc/python/recipe.rst

```diff
@@ -18,11 +18,23 @@ Recipe
     Describe a model by describing the cells and network, without any information about how the model is to be represented or executed.
 
     All recipes derive from this abstract base class.
-
+    
     Recipes provide a cell-centric interface for describing a model.
     This means that model properties, such as connections, are queried using the global identifier ``gid`` of a cell.
     In the description below, the term ``gid`` is used as shorthand for the cell with global identifier.
 
+    **Required Constructor**
+
+    The constructor must be implemented and call the base class constructor, as at the moment there is no way
+    to instruct Python to do that automatically. This ensures correct initialization of memory in the C++ class.
```

Comment:
and the user asks: C++? This is Python?! What's going on?

In all seriousness, add a note here reiterating the point that pyarb is a thin wrapper around C++ Arbor.
Then: Due to the py<->cxx binding process omitting this leads to memory corruption.